mlflow/langchain/__init__.py

```diff
@@ -906,10 +906,10 @@ def _load_model_from_local_fs(local_model_path, model_config_overrides=None):
             MODEL_CODE_PATH, flavor_conf.get(MODEL_CODE_PATH, None)
         )
         code_path = os.path.join(
-            local_model_path,
+            model_code_path,
```

Comment:
sorry, I meant rename `code_path` variable name in L908, not the variable `local_model_path` in L909

Reply:
do you mean like this?
```
model_code_path = os.path.join(
            local_model_path,
            os.path.basename(flavor_code_path),
        )

model = _load_model_code_path(
                model_code_path, {**(model_config or {}), **(model_config_overrides or {})}
            )
```